qa/suites/rados/thrash/workloads/pool-snaps-few-objects-redelete.yaml

```diff
@@ -0,0 +1,23 @@
+overrides:
```

Comment:
Also with this test:

description: rados/thrash/{0-size-min-size-overrides/3-size-2-min-size 1-pg-log-overrides/short_pg_log
    2-recovery-overrides/{more-async-partial-recovery} 3-scrub-overrides/{max-simultaneous-scrubs-5}
    backoff/peering ceph clusters/{fixed-4 openstack} crc-failures/default d-balancer/crush-compat
    mon_election/connectivity msgr-failures/few msgr/async-v2only objectstore/bluestore-comp-lz4
    rados supported-random-distro$/{ubuntu_latest} thrashers/careful thrashosds-health
    workloads/pool-snaps-few-objects-redelete}

/a/yuriw-2024-04-11_17:03:54-rados-wip-yuri6-testing-2024-04-02-1310-distro-default-smithi/7652491
```
2024-04-12T01:38:22.133 INFO:tasks.ceph.osd.7.smithi169.stderr:./src/osd/PG.cc: 1901: FAILED ceph_assert(!bad || !cct->_conf->osd_debug_verify_cached_snaps)
2024-04-12T01:38:22.133 INFO:tasks.ceph.osd.7.smithi169.stderr:
2024-04-12T01:38:22.134 INFO:tasks.ceph.osd.7.smithi169.stderr: ceph version 19.0.0-2838-ga5074d45 (a5074d4516d566e9d8b6aec912f26afd099de101) squid (dev)
2024-04-12T01:38:22.134 INFO:tasks.ceph.osd.7.smithi169.stderr: 1: (ceph::__ceph_assert_fail(char const*, char const*, int, char const*)+0x118) [0x55691a976362]
2024-04-12T01:38:22.134 INFO:tasks.ceph.osd.7.smithi169.stderr: 2: ceph-osd(+0x3f6519) [0x55691a976519]
2024-04-12T01:38:22.134 INFO:tasks.ceph.osd.7.smithi169.stderr: 3: ceph-osd(+0x38f81c) [0x55691a90f81c]
2024-04-12T01:38:22.134 INFO:tasks.ceph.osd.7.smithi169.stderr: 4: (PeeringState::Active::react(PeeringState::AdvMap const&)+0x19e) [0x55691ad92e5e]
2024-04-12T01:38:22.134 INFO:tasks.ceph.osd.7.smithi169.stderr: 5: ceph-osd(+0x840811) [0x55691adc0811]
2024-04-12T01:38:22.134 INFO:tasks.ceph.osd.7.smithi169.stderr: 6: (PeeringState::advance_map(std::shared_ptr<OSDMap const>, std::shared_ptr<OSDMap const>, std::vector<int, std::allocator<int> >&, int, std::vector<int, std::allocator<int> >&, int, PeeringCtx&)+0x266) [0x55691ad596e6]
2024-04-12T01:38:22.134 INFO:tasks.ceph.osd.7.smithi169.stderr: 7: (PG::handle_advance_map(std::shared_ptr<OSDMap const>, std::shared_ptr<OSDMap const>, std::vector<int, std::allocator<int> >&, int, std::vector<int, std::allocator<int> >&, int, PeeringCtx&)+0xfb) [0x55691ab7a76b]
2024-04-12T01:38:22.134 INFO:tasks.ceph.osd.7.smithi169.stderr: 8: (OSD::advance_pg(unsigned int, PG*, ThreadPool::TPHandle&, PeeringCtx&)+0x39c) [0x55691aaf318c]
2024-04-12T01:38:22.134 INFO:tasks.ceph.osd.7.smithi169.stderr: 9: (OSD::dequeue_peering_evt(OSDShard*, PG*, std::shared_ptr<PGPeeringEvent>, ThreadPool::TPHandle&)+0x237) [0x55691ab04957]
2024-04-12T01:38:22.134 INFO:tasks.ceph.osd.7.smithi169.stderr: 10: (ceph::osd::scheduler::PGPeeringItem::run(OSD*, OSDShard*, boost::intrusive_ptr<PG>&, ThreadPool::TPHandle&)+0x51) [0x55691ad374f1]
2024-04-12T01:38:22.134 INFO:tasks.ceph.osd.7.smithi169.stderr: 11: (OSD::ShardedOpWQ::_process(unsigned int, ceph::heartbeat_handle_d*)+0xab3) [0x55691ab0e6e3]
2024-04-12T01:38:22.134 INFO:tasks.ceph.osd.7.smithi169.stderr: 12: (ShardedThreadPool::shardedthreadpool_worker(unsigned int)+0x293) [0x55691b0024e3]
2024-04-12T01:38:22.135 INFO:tasks.ceph.osd.7.smithi169.stderr: 13: ceph-osd(+0xa82a44) [0x55691b002a44]
2024-04-12T01:38:22.135 INFO:tasks.ceph.osd.7.smithi169.stderr: 14: /lib/x86_64-linux-gnu/libc.so.6(+0x94b43) [0x7f9933f96b43]
2024-04-12T01:38:22.135 INFO:tasks.ceph.osd.7.smithi169.stderr: 15: /lib/x86_64-linux-gnu/libc.so.6(+0x126a00) [0x7f9934028a00]
```

The fix for https://tracker.ceph.com/issues/64347 is already included in the test branch, so this seems like a new crash related to this particular test.